src/containers/pages/IRS/JurisdictionsReport/index.tsx

```diff
@@ -191,9 +192,9 @@ const JurisdictionReport = (props: GenericJurisdictionProps & RouteComponentProp
     data,
     defaultPageSize: data.length,
     extraCellProps: { urlPath: baseURL },
-    getTdProps: (state: any, rowInfo: any, column: any, instance: any) => {
+    getTdProps: (_: any, rowInfo: any, column: any) => {
       return {
-        onClick: (e: any, handleOriginal: any) => {
+        onClick: (__: any, handleOriginal: any) => {
```

Comment:
Let's either define these Typings so they're not `any`s or let's open a ticket to do so, please

Reply:
Created the issue: https://github.com/onaio/reveal-frontend/issues/681